python/vyos/ethtool.py

```diff
@@ -101,8 +101,9 @@ def __init__(self, ifname):
         self._features = loads(out)[0]
 
         # Get information about NIC ring buffers
-        out, _ = popen(f'ethtool --json --show-ring {ifname}')
-        self._ring_buffer = loads(out)[0]
+        out, err = popen(f'ethtool --json --show-ring {ifname}')
```

Comment:
My only concern here: can we differentiate between an expected failure for Xen NICs and similar and an unexpected failure where `--show-ring` is _supposed_ to work? This approach may hide bugs.

I'm not against merging it as an immediate fix.

Reply:
It depends on what is "error" from our point of view:
- unexpected `ethtool` output
- a feature is not supported by a NIC
- both

I have an internal feeling that for our purposes a reason why it failed does not matter, but input from someone deeply familiar with interface config logic is welcomed.